sql/2021/privacy/most_common_cmps_for_iab_tcf_v2.sql

```diff
@@ -0,0 +1,45 @@
+#standardSQL
+# Counts of CMPs using IAB Transparency & Consent Framework
+# cf. https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#tcdata
+
+WITH pages_privacy AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    JSON_VALUE(payload, "$._privacy") AS metrics
+  FROM
+    `httparchive.pages.2021_07_01_*`
+),
+
+total_nb_pages AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    COUNT(0) AS total_nb_pages
+  FROM
+    `httparchive.pages.2021_07_01_*`
+  GROUP BY
+    1
+),
+
+pages_iab_tcf_v2 AS (
+  SELECT
+    client,
+    JSON_QUERY(metrics, "$.iab_tcf_v2.data") AS metrics
+  FROM
+    pages_privacy
+  WHERE
+    JSON_QUERY(metrics, "$.iab_tcf_v2.data") IS NOT NULL
+)
```

Comment:
I guess you are aiming for consistent use of the `pages_privacy` table, but seems these are fairly simple and we could simplify by merging the first and third table (this feedback applies throughout):

```suggestion
WITH pages_iab_tcf_v2 AS (
  SELECT
    _TABLE_SUFFIX AS client,
    JSON_QUERY(metrics, "$._privacy.iab_tcf_v2.data") AS metrics
  FROM
    `httparchive.pages.2021_07_01_*`
  WHERE
    JSON_QUERY(metrics, "$._privacy.iab_tcf_v2.data") IS NOT NULL
),

total_nb_pages AS (
  SELECT
    _TABLE_SUFFIX AS client,
    COUNT(0) AS total_nb_pages
  FROM
    `httparchive.pages.2021_07_01_*`
  GROUP BY
    1
),
```

Reply:
Fixed in `c642950`